<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="param314561401" failures="1" tests="1" name="LoginTest" time="0.722" errors="0" timestamp="27 Apr 2018 11:45:08 GMT">
  <testcase classname="com.newTours.demoaut.TestCases.LoginTest" name="@AfterMethod afterMethod" time="0.0">
    <failure type="java.lang.NullPointerException" message="Source must not be null">
      <![CDATA[java.lang.NullPointerException: Source must not be null
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:630)
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:606)
	at com.newtours.demoaut.util.Utility.catchExceptions(Utility.java:134)
	at com.newTours.demoaut.TestCases.BasePageSetUp.afterMethod(BasePageSetUp.java:136)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="com.newTours.demoaut.TestCases.LoginTest" name="LoginFunctionalityTest" time="0.031">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.newTours.demoaut.TestCases.LoginTest.LoginFunctionalityTest(LoginTest.java:27)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:514)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- LoginFunctionalityTest -->
</testsuite> <!-- LoginTest -->
